Mathlib/Algebra/Group/Fin/Basic.lean

```diff
@@ -39,12 +39,30 @@ instance addCommMonoid (n : ℕ) [NeZero n] : AddCommMonoid (Fin n) where
   nsmul := nsmulRec
   __ := Fin.addCommSemigroup n
 
-instance instAddMonoidWithOne (n) [NeZero n] : AddMonoidWithOne (Fin n) where
+namespace NatCast
```

Comment:
`Fin.NatCast.instAddMonoidWithOne` seems like a slightly strange isntance name to me; you could move this below the `def` and I think the `scoping` would still work as expected.

Reply:
Fixed.